src/software/thunderscope/gl/layers/gl_max_dribble_layer.py

```diff
@@ -0,0 +1,76 @@
+from pyqtgraph.opengl import *
+
+from software.thunderscope.constants import Colors, DepthValues
+from software.thunderscope.thread_safe_buffer import ThreadSafeBuffer
+from software.thunderscope.gl.layers.gl_layer import GLLayer
+from software.thunderscope.gl.graphics.gl_circle import GLCircle
+from software.thunderscope.gl.graphics.gl_polygon import GLPolygon
+import software.python_bindings as tbots_cpp
+from proto.import_all_protos import *
+
+from pyqtgraph.Qt import QtGui
+
+from software.thunderscope.gl.helpers.observable_list import ObservableList
+
+import math
+
+class GLMaxDribbleLayer(GLLayer):
+    """GLLayer"""
+
+    def __init__(self, name: str, buffer_size:int = 5) -> None:
+        """Initialize the GLMaxDribbleLayer
+
+        :param name: The displayed name of the layer
+        :param buffer_size: The buffer size, set higher for smoother plots.
+                            Set lower for more realtime plots. Default is arbitrary
+                            """
+        super().__init__(name)
+        self.setDepthValue(DepthValues.BACKGROUND_DEPTH)
+
+        self.world_buffer = ThreadSafeBuffer(buffer_size, World)
+
+        self.color = QtGui.QColor(255, 0, 0, 127)
+
+        self.dribble_radius_graphic = GLPolygon(parent_item = self, outline_color = self.color)
+        self.dribble_circle_graphic = GLCircle(parent_item = self, outline_color = self.color)
+        self.dribble_circle_graphic.set_radius(1, 24)
+
+    def refresh_graphics(self) -> None:
+        """Update graphics in this layer"""
+
+        world = self.world_buffer.get(block=False)
+        if world.HasField("dribble_displacement"):
+            dribbleDisp = world.dribble_displacement
```

Comment:
python variables use snake_case in our conventions so this should be `dribble_disp`